tools/config.ts

```diff
@@ -88,11 +88,11 @@ export const PROD_DEPENDENCIES = PROD_NPM_DEPENDENCIES.concat(APP_ASSETS);
 const SYSTEM_CONFIG_DEV = {
   defaultJSExtensions: true,
   paths: {
-    'main': `${APP_ROOT}main`,
```

Comment:
You can use ES2015 computed property name feature here I guess.
`[BOOTSTRAP_MODULE]: '${APP_ROOT}${BOOTSTRAP_MODULE}'`